server/.golangci.yml

```diff
@@ -32,28 +32,28 @@ issues:
     - channels/store/storetest/mocks
   exclude-rules:
     - linters:
-      # ignore unused warnings from enterprise code
```

Comment:
Can you please revert these unrelated changes?

Reply:
Done. Thank you.